frontend/components/views/legal/TermsOfService.vue

```diff
@@ -114,10 +114,10 @@
           assigns damages as a result of a dispute arising from the sale.</p>
 
         <p>
-          Subscription services will provide features such as notification of artist availability, but will not be
-          warranted for any particular use, and are provided AS IS. No guarantee is given that artists will accept
-          orders placed by subscription holders, or that artists will be timely in their workload management or
-          responses to subscription holders. Each Artist is responsible for their own performance.
+          Subscription services will be billed monthly, or at another explicitly agreed upon interval. The specific
```

Comment:
This is the meat of the changes here-- if you sign up for one of the new subscriptions we're offering but don't pay for services rendered, we'll be allowed to limit/suspend your account.